modules/030-cloud-provider-yandex/values.yaml

```diff
@@ -0,0 +1,528 @@
+cloudProviderYandex:
+  additionalExternalNetworkIDs: []
+  internal: {}
+global:
```

Comment:
Is it added by accident?

Reply:
Also, values.yaml is not needed here, there is openapi/config-values.yaml for defaults.

Reply:
Yeah, sry, removed